catalog/internal_tables.go

```diff
@@ -120,19 +176,12 @@ var InternalTables = struct {
 		ValueColumns: []string{"position"},
 		DDL:          "channel TEXT PRIMARY KEY, position TEXT",
 	},
-	PgReplicationLSN: InternalTable{
-		Schema:       "__sys__",
-		Name:         "pg_replication_lsn",
-		KeyColumns:   []string{"slot_name"},
-		ValueColumns: []string{"lsn"},
-		DDL:          "slot_name TEXT PRIMARY KEY, lsn TEXT",
-	},
 	PgSubscription: InternalTable{
 		Schema:       "__sys__",
 		Name:         "pg_subscription",
-		KeyColumns:   []string{"name"},
-		ValueColumns: []string{"connection", "publication"},
-		DDL:          "name TEXT PRIMARY KEY, connection TEXT, publication TEXT",
+		KeyColumns:   []string{"subname"},
+		ValueColumns: []string{"subconninfo", "subpublication", "subskiplsn", "subenabled"},
```

Comment:
The `sub` prefix for all column names can be removed.

Reply:
`sub` is chosen to maintain consistency with the [pg_subscription catalog in PostgreSQL](https://www.postgresql.org/docs/current/catalog-pg-subscription.html).